File: game-server/src/com/aionemu/gameserver/configs/main/PetFeedUnusualStorageArtifactCaptureConfig.java
package com.aionemu.gameserver.configs.main;

import com.aionemu.commons.configuration.Property;

/**
 * Disabled-by-default config shell for pet-feed unusual-storage parity artifacts.
 */
public class PetFeedUnusualStorageArtifactCaptureConfig {

	@Property(key = "gameserver.petfeed.unusual_storage_artifacts.enabled", defaultValue = "false")
	public static boolean ENABLED;

	@Property(key = "gameserver.petfeed.unusual_storage_artifacts.output_dir", defaultValue = "./parity-artifacts/pet-feed-unusual-storage/java")
	public static String OUTPUT_DIR;

	@Property(key = "gameserver.petfeed.unusual_storage_artifacts.max_pending_contexts_per_player", defaultValue = "4")
	public static int MAX_PENDING_CONTEXTS_PER_PLAYER;

	@Property(key = "gameserver.petfeed.unusual_storage_artifacts.max_queued_artifacts", defaultValue = "32")
	public static int MAX_QUEUED_ARTIFACTS;

	@Property(key = "gameserver.petfeed.unusual_storage_artifacts.allowed_scenario", defaultValue = "pet_feed_unusual_storage")
	public static String ALLOWED_SCENARIO;
}

File: dotnetConversion/src/Aion.GameServer/Configs/Main/PetFeedUnusualStorageArtifactCaptureConfig.cs
namespace Aion.GameServer.Configs.Main;

/// <summary>Java parity: configs/main/PetFeedUnusualStorageArtifactCaptureConfig.</summary>
public static class PetFeedUnusualStorageArtifactCaptureConfig
{
    /// <summary>Property key: gameserver.petfeed.unusual_storage_artifacts.enabled</summary>
    public static bool ENABLED = false;

    /// <summary>Property key: gameserver.petfeed.unusual_storage_artifacts.output_dir</summary>
    public static string OUTPUT_DIR = "./parity-artifacts/pet-feed-unusual-storage/java";

    /// <summary>Property key: gameserver.petfeed.unusual_storage_artifacts.max_pending_contexts_per_player</summary>
    public static int MAX_PENDING_CONTEXTS_PER_PLAYER = 4;

    /// <summary>Property key: gameserver.petfeed.unusual_storage_artifacts.max_queued_artifacts</summary>
    public static int MAX_QUEUED_ARTIFACTS = 32;

    /// <summary>Property key: gameserver.petfeed.unusual_storage_artifacts.allowed_scenario</summary>
    public static string ALLOWED_SCENARIO = "pet_feed_unusual_storage";
}
